src/Resources/config/services.yaml

```diff
@@ -28,6 +28,12 @@ services:
             - '@security.token_storage'
             - !tagged_iterator lle_dashboard.widget
 
+    Lle\DashboardBundle\Service\WidgetCompacterService:
+        class: Lle\DashboardBundle\Service\WidgetCompacterService
+        public: true
```

Comment:
you don't need to make it public. Public means you can directly access it from the container.

Reply:
class isn't required because you already wrote it above (as the key)